core/src/avm2/globals/flash/system/capabilities.rs

```diff
@@ -0,0 +1,33 @@
+//! `flash.display.Capabilities` native methods
+
+use crate::avm2::{Activation, AvmString, Error, Object, Value};
+
+/// Implements `flash.system.Capabilities.version`
+pub fn get_version<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    // TODO: Report the correct OS instead of always reporting Linux
+    Ok(AvmString::new_utf8(
+        activation.context.gc_context,
+        format!("LNX {},0,0,0", activation.avm2().player_version),
+    )
+    .into())
+}
+
+/// Implements `flash.system.Capabilities.playerType`
+pub fn get_player_type<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    _this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    // TODO: When should "External" be returned?
+    let player_type = if cfg!(target_family = "wasm") {
+        "PlugIn"
+    } else {
+        "StandAlone"
+    };
```

Comment:
Would it be a bad idea or helpful to insert this link as a comment?:

https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/system/Capabilities.html#playerType

Or is it implied?

Reply:
(I am not certain, but, I assume that string literals are fine to use here, since I would assume that the values are only used in one place in the codebase and only used by ActionScript applications).

Reply:
> Would it be a bad idea or helpful to insert this link as a comment?

We don't usually do that and there's no particular reason to do that here